extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -173,6 +173,10 @@ function Autoplay({
   play,
   pause,
 }) {
+  // TODO(alanorozco): We can't have multiple useStyles here,
+  // so maybe this component should be autoplay.js
+  const classes = useStyles();
```

Comment:
why can't we have multiple use styles?

Reply:
Unsure if the transform makes assumptions/looks for calls named exactly `useStyles`. (If we use more than one import that exports a `useStyles`, it would have to be renamed.)

Reply:
(also sorry for being unclear: by multiple `useStyles` I meant multiple imports of different functions named the same, not multiple calls to the same import)

Reply:
Ah! It should work fine, since we actually don't transform the preact component files at all. The only transformed files are the `.jss` ones